content/posts/mongo_db_databases.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Introducing MongoDB databases"
+date: 2024-10-14
+tags:
+    - News
+author: Costin Sin
+linkedIn: https://www.linkedin.com/in/costin-sin/
+thumbnail: /images/ferretdb-genezio.webp
+preview: Genezio users can now set up MongoDB databases! You can now easily manage your MongoDB databases alongside your serverless functions. Learn more about this new feature.
```

Comment:
Learn more about this new feature. --> Learn how to deploy scalable, secure NoSQL databases effortlessly!